examples/hacker_news_pwa/lib/src/feed_component.html

```diff
@@ -14,7 +14,15 @@
   <span class="rank">{{i + startingRank}}</span>
   <item [item]="item"></item>
 </article>
-<div>
-  <a [routerLink]="nextPageUrl">Next page</a>
-</div>
 </template>
+
+<footer>
+  <nav>
+  <div *ngIf="previousPageUrl != null"  >
```

Comment:
Can't. The divs split the view left/right. They are styled as `flex`. You don't want the `a` tags as flex because then their clickable size extends beyond the text.

And in the case where there is only one next/previous the clickable area in the whole width of the footer...

Reply:
Fair enough!

Reply:
nit: if you're keeping them, mind indenting them please?

Reply:
done